cmd/list_groups.go

```diff
@@ -13,8 +13,8 @@ func ListGroupValidations(cfg uaa.Config) error {
 	return nil
 }
 
-func ListGroupsCmd(gm uaa.GroupManager, printer cli.Printer, filter, sortBy, sortOrder, attributes string, startIndex, count int) error {
```

Comment:
It looks like we have removed the ability to paginate the client/ groups / users endpoint. While I think this is convenient, we have concerns about performance as we know of some customers that have ~300,000 users. 

We think that a nice compromise would be to keep this logic, however perhaps add the ability to 'opt-in' to listing *all* the users/groups/clients. (Perhaps a flag named `--all`). However, if this flag is not provided, then fallback to using pagination. Can we also add back `startIndex` and `count`?